src/Quasar/Mount.purs

```diff
@@ -47,10 +48,10 @@ data MountConfig
 
 instance showMountConfig ∷ Show MountConfig where
   show (ViewConfig { query, vars })
-    = "(ViewConfig { query: " <> show query
+    = "(ViewConfig { query: " <> Sql.printQuery query
     <> ", vars: " <> show vars <> " })"
   show (ModuleConfig config)
-    = "(ModuleConfig { module: " <> config."module" <> " })"
+    = "(ModuleConfig { module: " <> Sql.printModule config."module" <> " })"
```

Comment:
Is there no show instance for these? Not that it matters too much, since this is debuggy code, but usually you `show` all the way down.

Reply:
no Show instance